modules/ROOT/pages/troubleshoot-guide.adoc

```diff
@@ -5,24 +5,43 @@ endif::[]
 
 This topic describes common errors and steps to resolve them when installing Anypoint Runtime Fabric.
 
+== Resuming a Failed Installation
+
 You can resume a failed installation at the point where it stopped by running the init script:
```

Comment:
You can resume an installation at the point where it failed...

`init` script

Reply:
Fixed!